src/_live-monitor.ts

```
@@ -94,12 +95,28 @@ export class LiveMonitor implements Disposable {
 					try: async () => readFile(file, 'utf-8'),
-					catch: () => new Error('File read failed'),
+					catch: (error) => {
+						// Preserve original error information for better debugging
+						if (error instanceof Error) {
+							return error;
+						}
+						return new Error(`File read failed: ${String(error)}`);
+					},
 				});
 
 				const fileResult = await fileReader();
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)